videoframes.py

```diff
@@ -79,6 +79,10 @@ def mouseReleaseEvent(self, event):
             else:
                 self.toggle_playback()
 
+    def mouseDoubleClickEvent(self, event):
+        """Toggles fullscreen mode on the current stream."""
+        self._fullscreen(self)
```

Comment:
Yes, perhaps even:

```python
if not event.modifiers() == (QtCore.Qt.ControlModifier | QtCore.Qt.ShiftModifier):
    self._fullscreen(self)
```
I was thinking that we can use the shift modifier for resizing frames.